src/mongoose_client_api_rooms_users.erl

```diff
@@ -51,10 +51,10 @@ from_json(Req, #{user := User,
                                               UserToInvite, <<"member">>),
     {true, Req2, State};
 from_json(Req, State) ->
-    mongoose_client_api_rooms:forbidden_request(Req, State).
+    mongoose_client_api:forbidden_request(Req, State).
```

Comment:
Separate `from_json` clause for `role_in_room := none` with 404?

Reply:
Good question, if we return 403 if user is not member and tries to post I think we should also return 403 in this case. If we choose to return 404 here, I think 404 should be returned in the other case as well.